clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -106,31 +106,39 @@ public void test_C_client_capabilities_exposed() throws Exception {
   public void test_D_server_capabilities_exposed() throws Exception {
     Capability[] capabilities = readTopology().getSingleStripe().getActiveServerEntity(ehcacheServerEntityIdentifier).get().getManagementRegistry().get().getCapabilities().toArray(new Capability[0]);
 
-    assertThat(capabilities.length).isEqualTo(5);
+    assertThat(capabilities.length).isEqualTo(6);
 
     assertThat(capabilities[0].getName()).isEqualTo("ClientStateSettings");
-    assertThat(capabilities[1].getName()).isEqualTo("PoolSettings");
-    assertThat(capabilities[2].getName()).isEqualTo("PoolStatistics");
-    assertThat(capabilities[3].getName()).isEqualTo("ServerStoreSettings");
-    assertThat(capabilities[4].getName()).isEqualTo("ServerStoreStatistics");
+    assertThat(capabilities[1].getName()).isEqualTo("ClusteredTierManagerSettings");
+    assertThat(capabilities[2].getName()).isEqualTo("PoolSettings");
+    assertThat(capabilities[3].getName()).isEqualTo("PoolStatistics");
+    assertThat(capabilities[4].getName()).isEqualTo("ServerStoreSettings");
+    assertThat(capabilities[5].getName()).isEqualTo("ServerStoreStatistics");
 
-    assertThat(capabilities[3].getDescriptors()).hasSize(4); // time descriptor + 3 dedicated store
+    assertThat(capabilities[4].getDescriptors()).hasSize(4); // time descriptor + 3 dedicated store
 
     // stats
 
-    assertThat(capabilities[4].getDescriptors()).containsOnlyElementsOf(SERVER_STORE_DESCRIPTORS);
-    assertThat(capabilities[2].getDescriptors()).containsOnlyElementsOf(POOL_DESCRIPTORS);
+    assertThat(capabilities[5].getDescriptors()).containsOnlyElementsOf(SERVER_STORE_DESCRIPTORS);
+    assertThat(capabilities[3].getDescriptors()).containsOnlyElementsOf(POOL_DESCRIPTORS);
 
     // ClientStateSettings
 
     assertThat(capabilities[0].getDescriptors()).hasSize(1);
     Settings settings = (Settings) capabilities[0].getDescriptors().iterator().next();
     assertThat((String[]) settings.get("attachedStores")).containsOnly("dedicated-cache-1", "shared-cache-2", "shared-cache-3");
 
-    // ServerStoreSettings
+    // ClusteredTierManagerStateSettings
+
+    assertThat(capabilities[1].getDescriptors()).hasSize(1);
+    settings = (Settings) capabilities[1].getDescriptors().iterator().next();
+    assertThat(settings.get("type")).isEqualTo("ClusteredTierManager");
+    assertThat(settings.get("defaultServerResource")).isEqualTo("primary-server-resource");
```

Comment:
@anthonydahanne : FYI, a new provider is responsible to expose the cluster tier manager now.
So `defaultServerResource ` is not anymore under `PoolSettings ` capability. `PoolSettings` will only expose things about pools now.